admin/Model/ModeStatusInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+
+namespace Pronko\LiqPayAdmin\Model;
```

Comment:
Please move this interface to Pronko\LiqPayAdmin\Api\Data or move constants into the source class.

Reply:
@mcspronko also if we are going to class admin/Model/Config/Source/Mode -> admin/Model/Config/Source/Environment  this interface better to rename EnvironmentStatusInterface ?